### ISSUE TITLE
ERROR: Cannot execute binary file

### ISSUE THREAD

**Issue by DavidGO464** [statgen/Minimac4#63]
Hello !!!

I wonder if minimac4 can be used in **Mac** computers Terminal or it is **restricted to** Linux OS 

![Screenshot 2023-08-05 at 4 34 26 PM](https://github.com/statgen/Minimac4/assets/87618961/62d62e05-a08d-44a7-ba8d-c20ee1818b1e)

**Comment by jonathonl** [comment]
Yes, it can be used on Mac OS if compiled on Mac OS. Where did you get the minimac4 binary from?

**Comment by DavidGO464** [comment]
I first trid to used the following: 

<img width="672" alt="Screenshot 2023-08-07 at 2 39 32 PM" src="https://github.com/statgen/Minimac4/assets/87618961/da4fc70b-68ff-4602-bc2c-099e8ef4f958">

then It says that the inataller was not in the downloaded documents...
thus I tried the github version:


<img width="920" alt="Screenshot 2023-08-07 at 2 41 04 PM" src="https://github.com/statgen/Minimac4/assets/87618961/664caf93-6a52-4044-9bd9-e60d7035115b">
<img width="1230" alt="Screenshot 2023-08-07 at 2 41 16 PM" src="https://github.com/statgen/Minimac4/assets/87618961/e0b5303a-9828-4148-b7a8-6af1d2ed5bcb">


But still getting **"cannot execute binary file"**

how did you installed it?
(I am using macbook M2 ARM)

**Comment by jonathonl** [comment]
The prebuilt executable on the Github releases section is a Linux-only executable. You have to build from source (https://github.com/statgen/Minimac4#building-from-source) on MacOS. Unfortunately, recent versions of MacOS have broken compilation of some the dependencies, so it might be difficult for users not experienced with compiling c++ code. MacOS is not generally supported, but certainly possible if you know what you are doing.

For MacOS, it will look something like this:
```
# cget can be installed with pip3
cget ignore --prefix ./minimac4-install xz   # XZ is broken on recent versions of MacOS.
cget install --prefix ./minimac4-install statgen/Minimac4
./minimac4-install/bin/minimac4 --help
```
